src/pages/AdminForm.js

```diff
@@ -108,11 +108,14 @@ const AdminForm = props => {
 
         <SubmitButton>Create Application Group</SubmitButton>
       </form>
+
       {isDuplicate && (
-        <p>{`This Cohort Name already exists for ${props.cohortType}`}</p>
+        <Message color="red">
+          <Message.Header>Error Message</Message.Header>
+          <p>{`This Cohort Name already exists for ${props.cohortType}`}</p>
+        </Message>
       )}
```

Comment:
if i type a duplicate cohort name, and then get the error message "this This Cohort Name already exists for ${props.cohortType}" and then change the cohort type to one that the name doesnt exist for, I get a faulty message because it automatically changes. Can you make it so this message doesn't update more than once after the user submits a duplicated name and stays on page?

Reply:
otherwise the error message looks great!

Reply:
![Screenshot_2019-12-08 Create Application Form](https://user-images.githubusercontent.com/38357539/70401511-e9e8f480-19fd-11ea-93c1-d140c3e0b5da.png)